src/client/Client.h

```diff
@@ -934,6 +934,8 @@ class Client : public Dispatcher, public md_config_obs_t {
   bool check_quota_condition(Inode *in, const UserPerm& perms,
 			     std::function<bool (const Inode &)> test);
   bool is_quota_files_exceeded(Inode *in, const UserPerm& perms);
+  bool is_quota_bytes_exceeded(Inode *in, int64_t new_bytes,
```

Comment:
hmm, I think you intended to override the is_quota_files_exceeded() method, right?

Reply:
sorry, It's fixed.